### ISSUE TITLE
Captcha Alarm not going off

### ISSUE THREAD

**Issue by xAkumu** [JohnnyRS/PandaCrazy-Max#49]
I can't get my captcha alarm to work and it's really slowed me down. I've tried clearing my cache/cookies, reinstalling, and turning the alarm off and back on and nothing is working.

**Comment by JohnnyRS** [comment]
Was the alarm working before? I changed a few things trying to detect captcha with the last update. I am not able to test it on my system.

**Comment by xAkumu** [comment]
Yes it worked before. It stopped working after I cleared my cache and
cookies when I cleaned up my system.

**Comment by seasons1981** [comment]
yeah, no sound or popup so we have no idea when we have one, unless you are working on a batch and know there are more available you would never know.

**Comment by JohnnyRS** [comment]
Well if it worked before the new update I will revert the changes I made to the captcha detection to the previous update. I got a report that it wasn't working so I changed it. Also if someone that gets captchas can show me a screenshot of what is shown on the page with a captcha it could help me make the detection better. Thanks.

**Comment by seasons1981** [comment]
as in on the page we put our captcha in or what kind of information do you need?  I am more than willing to help you out with this, I just am not sure what screenshot would be needed. (plus there aren't many batches up that are worth it right now, so it is harder to get those captchas to come in)

**Comment by JohnnyRS** [comment]
Right now it checks to see if the url it gets back from a panda doesn't have assignment_id= at the end and it doesn't get an error message than it assumes it probably is a captcha. What might be better is to have it look for a unique text on the page so it knows for sure. Lot of the screenshots of mturk captchas are from years ago so I want to see a screenshot of the text that tells the user to fill in the captcha because it might haven changed now.

**Comment by xAkumu** [comment]
<img width="765" alt="928249699111d477bf0e3e7087a94519" src="https://user-images.githubusercontent.com/91992865/136445650-8621e5dd-53eb-4dc8-bf3c-89a1e04ac014.png">


heres a picture of the captcha, the alarm still isn't working

**Comment by JohnnyRS** [comment]
Thanks. That's what I wanted. I'll see if I can get it to work in the next update.

**Comment by JohnnyRS** [comment]
I updated the extension with a fix for the captchas and it looks like it's already live on chrome store. Should update next time you start up chrome. Hope it works.

**Comment by xAkumu** [comment]
Thank you so much! Not only does the alarm go off now, I can do captchas in Panda Crazy now too.
I really appreciate it <3
